integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/End2EndTestITStep.java

```diff
@@ -251,4 +254,16 @@ public void testMetrics() {
         assertThat(ingressMetrics).contains("http_server_requests_seconds_count{method=\"POST\",outcome=\"SUCCESS\",status=\"200\",uri=\"/events/plain\",} 1.0");
     }
 
+    @After
+    public void cleanUp() {
+        if (IntStream.range(0, getBridgeList().getItems().size()).anyMatch(b -> getBridgeList().getItems().get(b).getId().equals(bridgeId))) {
+            if (listProcessors(bridgeId).getSize() > 0) {
```

Comment:
Maybe I have a suggestion for improvement. Would it be possible to go through the list of processors and delete all processors? I know that currently we have only test which use only one processor, but it may change in the future and it is necessary to delete all processors to delete the bridge.

Reply:
No, the tests can run on demo cluster. In such case we can delete only resources which we created, nothing else.

Reply:
Understand but I meant only processors in the given bridge which we are deleting. From what I've experienced it is not possible to delete bridge without also deleting all processors.

Reply:
Understood i can add loop for delete multiple processor